### PR TITLE
Improve French translation

### PR DESCRIPTION
**Pull request by neomilium**
Thanks @mtancoigne we now have a good base to improve internationalization and provide a full translation.

This PR contains only French translation fixes.

### CODE REVIEW

**Comment by mtancoigne** [comment]
Wow ! That's a lot of good changes ! I'll merge it tonight.

**Comment by mtancoigne** [comment]
Can you tell me when your work is over so I can merge it ? There is too much changes in files that doesn't exist anymore to do a quick rebase :)

File: config/locales/devise.fr.yml [hunk 1]
```diff
@@ -11,15 +11,15 @@ fr:
...
+      already_authenticated: "Vous êtes déjà autentifé(e)."
+      inactive: "Votre compte n’est pas encore activé."
       invalid: "%{authentication_keys} ou mot de passe invalide."
       locked: "Votre compte est vérouillé."
       last_attempt: "Il vous reste un essai avant le verouillage de votre compte."
       not_found_in_database: "%{authentication_keys} ou mot de passe invalide."
-      timeout: "Votre session a expiré. Veuillez vous reconnecter pour continuer."
-      unauthenticated: "Vous devez être indentifié·e pour pouvoir continuer."
-      unconfirmed: "Vous devez valider votre adresse email pour pouvoir continuer."
+      timeout: "Votre session a expirée. Veuillez vous reconnecter pour continuer."
```
Comment (mtancoigne) [inline config/locales/devise.fr.yml]: It should be `Votre session a expiré` or `Votre session est expirée`, but not `Votre session a expirée`

**Comment by mtancoigne** [comment]
I started to merge all your changes in my PR; when it's done, please update your code with mine, as merging this is not really easy.

**Comment by mtancoigne** [comment]
Thanks for the ixes, they are now merged in the other PR.

File: config/locales/devise.fr.yml (hunk 1 above)
Comment (neomilium) [reply config/locales/devise.fr.yml]: Right about language fix. About devise translation, we should definitely use the dedicated gem (devise-i18n).

**Comment by neomilium** [comment]
@manno This PR is now squashed and ready to be merged: it only contains French translation improvements (no core modifications).
Thanks

**Comment by mtancoigne** [comment]
Sadly, it's totally out of sync with #382...

**Comment by neomilium** [comment]
@mtancoigne: to ease process without blocking improvements, remove all my commits from your PR, I'll provide them separately.

**Comment by neomilium** [comment]
@manno I close this PR for now and will provide these enhancements (and further) on top of #385 when its merged.

**Pull request closed without merging**